src/Graph.tsx

```diff
@@ -315,14 +327,10 @@ function elkJSON(elkNode: StateElkNode): any {
   };
 }
 
-const GraphNode: React.FC<{ elkNode: StateElkNode }> = ({ elkNode }) => {
-  return <StateNodeViz stateNode={elkNode.node.data} node={elkNode.node} />;
-};
-
 export async function getElkGraph(
   rootDigraphNode: DirectedGraphNode,
 ): Promise<ElkNode> {
-  const rectMap = getRectMap();
+  const rectMap = await getRectMap(rootDigraphNode.id);
```

Comment:
any particular reason behind this change? I don't see it immediately when reading through those changes and it seems that it either is not needed or it has fixed some race condition - but it's hard to tell without knowing more context behind this

Reply:
Not a race condition; without this change, when switching between actors, `getElkGraph()` is called when the previous graph is destroyed and the new graph isn't yet rendered; therefore, `getRectMap(...)` is useless and calling `.width` was throwing an error (can't read `.width` of `undefined`).

This was introduced to fix that consistent error.